driver-mapping/pom.xml

```diff
@@ -108,11 +109,37 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${groovy.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is a test dependency. I wouldn't care so much about it, but I could live with a separate module if this is a concern. Do you think it could pollute our test classpath is some dangerous way?